shell/platform/windows/fixtures/main.dart

```diff
@@ -36,6 +37,46 @@ void hiPlatformChannels() {
   });
 }
 
+@pragma('vm:entry-point')
+void alertPlatformChannel() async {
+  // Serializers for data types are in the framework, so this will be hardcoded.
```

Comment:
If I am incorrect and we have a better way to serialize the data map, please correct me.

Reply:
/cc @gaaclarke Any thoughts?

If we stick to the current approach, please replace the type labels (like `_valueMap`, `_valueString`) with constants to improve readability